js102/index.html

```diff
@@ -55,21 +55,78 @@
 					<p>Slides: <a href="http://yonet.github.io/JS102/#/" target="_blank">bit.ly/JS102</a></p>
 				</section>
 				<!-- /Welcome -->
+
+				<!-- Tone Setting -->
+				<section>
+					<h2>Hello!</h2>
+					<p>Some 'guidelines' for today</p>
+					<ul>
+						<li>We are here to support your learning in a safe, inclusive environment</li>
```

Comment:
I really like these guidelines and wonder if we should update them in the intro slides